content/client-lib-development-guide/features.textile

```diff
@@ -300,6 +300,9 @@ h3(#rest-channel). Channel
 *** @(RSL6a2)@ A set of tests must exist to ensure that the client library provides interoperability for the @extras@ field which is a JSON-encodable object (ie a value that represents a JSON @object@ value and supports serialization to and from JSON text). The test, at a minimum, should publish a message with an @extras@ object such as @{"push":[{"title":"Testing"}]}@ and ensure it is received with an equivalent JSON-encodable object
 ** @(RSL6b)@ If, for example, incompatible encryption details are provided or invalid Base64 is detected in the message payload, an error message will be sent to the logger, but the message will still be delivered with last successful decoding and the @encoding@ field. For example, if a message had a decoding of "utf-8/cipher+aes-128-cbc/base64", and the payload was successfully Base64 decoded but the payload could not be decrypted because the @CipherParam@ details were not configured, the message would be delivered with a binary payload and an @encoding@ with the value "utf-8/cipher+aes-128-cbc". Additional steps need to be taken if decoding failed on "vcdiff" encoding; see "RTL18":#RTL18
 * @(RSL7)@ @Channel#setOptions@ takes a @ChannelOptions@ object and sets or updates the stored channel options, then indicates success
+* @(RSL8)@ @Channel#status@ function:
```

Comment:
I think the spec should include details about what request the library sends (eg, GET vs POST, which URI to use). `RSC16` (Rest#time) is an example of this, although it's actually kind of incorrect since it specifies a static URI but the request URI will change depending on environment (see RSC11b). Maybe there's a better example in the spec somewhere but it would take some time to check.

Reply:
I've expanded this description and also the description of RSL8a as the GET http request could potentially return an error.

Reply:
Looks good, although I would suggest changing it to something along the lines of:
> makes a http get request to @\<restHost\>/channels/\<channelId\>@ where @\<restHost\>@ represents the current rest host as described by "@RSC11@":#RSC11.

Also, although it's fairly obvious that it's passed in as an argument to `status`, I think we should explicitly state where \<channelId\> comes from just to avoid any ambiguity.

Reply:
I've just pushed the first suggestion in this comment.

Regarding your second suggestion. I wasn't expecting channelId to be passed into the channel.Status function at all. This is because Status is a method on a channel object that will already know the its own name, which is the channelID. 

If I have misunderstood something here, please feel free to use the suggestion tool to show me what you mean.